pkg/workspace/doc_extractor.go

```diff
@@ -37,6 +36,8 @@ func (v DocExtractor) Extract(annName structmeta.AnnotationName) ([]*yamlmeta.Do
 	return matchedDocs, nonMatchedDocs, nil
 }
 
+var ignoreUnknown = yamltemplate.MetasOpts{IgnoreUnknown: true}
```

Comment:
should we just inline this? seems a bit weird to have workspace package own this little tidbit as top level var.

Reply:
I mean, we can... I hear you about a package-level dangling var... I guess the fact that we're building the same value object again and again isn't too big a deal.

Reply:
your call. im ok either way with this.

> the fact that we're building the same value object again and again isn't too big a deal

yup, if we ever start optimizing ytt for perf i dont think this will show up on your radar.